aliyun/src/main/java/org/apache/iceberg/aliyun/AliyunProperties.java

```diff
@@ -50,7 +50,7 @@
    * For more information about how to obtain an AccessKey pair, see:
    * https://www.alibabacloud.com/help/doc-detail/53045.htm
    */
-  public static final String ACCESS_KEY_SECRET = "access.key.secret";
+  public static final String CLIENT_SECRET_ACCESS_KEY = "client.secret-access-key";
```

Comment:
In alibaba cloud, we often say the access key pair is composed by a `key` and a `secret`.  The `key` is similar to the `username`, and the `secret` is similar to the password. You can see the document [1] and [2].

So in my view, a better way is to rename it as `client.access-key-secret` to match the alibaba cloud. About the OSS Java SDK,   although they use a different way to name the variable but I think the document is more suitable & readable for most of the aliyun people.


[1]. https://www.alibabacloud.com/help/doc-detail/63482.htm
[2]. https://www.alibabacloud.com/help/doc-detail/53045.htm

Reply:
that makes sense, I will update the PR